bin/docserv-createconfig

```diff
@@ -118,7 +119,10 @@ def generate_config(languages, default_lang, git_url, branches, version="0.9.10"
     etree.SubElement(maintainers, "contact").text = FIXME if not contact else contact
 
     for language in languages:
-        description = etree.SubElement(root, "desc", default="0", lang=language)
+        is_default_lang=0
+        if language == default_lang:
+            is_default_lang = 1
```

Comment:
You could also done that as one-liner (it's not really necessary, just an idea):

```python
is_default_lang = 1 if language == default_lang else 0
```

Reply:
To be honest, i think the original version is easier to read.